.github/workflows/build-image.yml

```diff
@@ -18,6 +19,23 @@ jobs:
   build-image:
     runs-on: ubuntu-latest
 
+    name: build-image (${{ name }})
+
+    strategy:
```

Comment:
@renchap apparently the way to build multiple images is to use a matrix, idk how I feel about this / what this change could potentially break. (e.g., required actions)